rott/vgatext.c

```diff
@@ -180,7 +183,12 @@ int vgatext_main(SDL_Window *window, Uint16 *screen)
 			next += BLINK_HZ;
 		}
 
-		SDL_UpdateTexture(texture, NULL, (*windowsurface)->pixels, (*windowsurface)->pitch);
+		if (SDL_LockTexture(texture, NULL, &dst_pixels, &dst_pitch) == 0)
```

Comment:
I don't think we need this change here. This code is absolutely not performance sensitive and the `SDL_UpdateTexture()` solution is still easier to read.